sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/Docs/ConversationAnalysisClient.xml

```diff
@@ -2,196 +2,196 @@
+
+RequestContent content = RequestContent.Create(new
+{
+    analysisInput = new
+    {
+        conversationItem = new
+        {
             id = "<id>",
             participantId = "<participantId>",
         },
     },
-    parameters = new {
+    parameters = new
+    {
         projectName = "<projectName>",
         deploymentName = "<deploymentName>",
     },
     kind = "Conversation",
-};
-
-Response response = await client.AnalyzeConversationAsync(RequestContent.Create(data));
+});
+Response response = await client.AnalyzeConversationAsync(content);
 
 JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
 Console.WriteLine(result.GetProperty("kind").ToString());
 ]]></code>
-This sample shows how to call AnalyzeConversationAsync with all request content, and how to parse the result.
+This sample shows how to call AnalyzeConversationAsync with all request content and parse the result.
 <code><![CDATA[
-var credential = new AzureKeyCredential("<key>");
-var endpoint = new Uri("<https://my-service.azure.com>");
-var client = new ConversationAnalysisClient(endpoint, credential);
-
-var data = new {
-    analysisInput = new {
-        conversationItem = new {
-            text = "<text>",
+Uri endpoint = new Uri("<endpoint>");
+AzureKeyCredential credential = new AzureKeyCredential("<key>");
+ConversationAnalysisClient client = new ConversationAnalysisClient(endpoint, credential);
+
+RequestContent content = RequestContent.Create(new
+{
+    analysisInput = new
+    {
+        conversationItem = new
+        {
             id = "<id>",
             participantId = "<participantId>",
             language = "<language>",
             modality = "transcript",
             role = "agent",
         },
     },
-    parameters = new {
+    parameters = new
+    {
         projectName = "<projectName>",
         deploymentName = "<deploymentName>",
         verbose = true,
         isLoggingEnabled = true,
         stringIndexType = "Utf16CodeUnit",
         directTarget = "<directTarget>",
-        targetProjectParameters = new {
-            key = new {
+        targetProjectParameters = new
+        {
+            key = new
+            {
                 query = "<query>",
-                callingOptions = new {
-                    verbose = true,
-                    log = true,
-                    show_all_intents = true,
-                    timezoneOffset = 123.45f,
-                    spellCheck = true,
-                    bing_spell_check_subscription_key = "<bing-spell-check-subscription-key>",
+                callingOptions = new Dictionary<string, object>()
+                {
+                    ["verbose"] = true,
+                    ["log"] = true,
+                    ["show-all-intents"] = true,
+                    ["timezoneOffset"] = 123.45F,
+                    ["spellCheck"] = true,
+                    ["bing-spell-check-subscription-key"] = "<bing-spell-check-subscription-key>",
```

Comment:
This doesn't seem right. There is a model there, and you get IntelliSense that way. Why are you creating as a dictionary with no IntelliSense support?

Reply:
Because if you notice this sample code is for a protocol method therefore we do not use models in samples like this.

Reply:
Apologies I have to dismiss your change request to unblock other possible changes from the generator.
Please feel free to ping me if you have other concerns on this, I could explain in details.